systest/backup/common/utils.go

```diff
@@ -91,6 +102,221 @@ func CopyToLocalFs(t *testing.T) {
 	require.NoError(t, testutil.DockerCp(srcPath, copyBackupDir))
 }
 
+func RemoveContentsOfPerticularDir(t *testing.T, dir string) {
+	d, err := os.Open(dir)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer d.Close()
+	names, err := d.Readdirnames(-1)
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, name := range names {
+		err = os.RemoveAll(filepath.Join(dir, name))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+	}
+}
+
+func DeleteNamespace(t *testing.T, id uint64, jwtToken string, whichAlpha string) {
```

Comment:
We have a `testutil.DeleteNamespace`. Please check for other functions too in the testutil library.

Reply:
Resolved.

1] Using testutil.DeleteNamespace()
2] Removed CheckSchemaExists(), now using AssertGetGQLSchema().
3] Removed changeFolderPermission(), Handling folder deletion from the container.